### PR TITLE
Nt splah page

### PR DESCRIPTION
**Pull request by TheHebi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,31 +1,71 @@
...
+            <option value="audio">Audio</option>
+            <option value="photos">Photos</option>
+            <option value="amuscripts">Manuscripts</option>
+            <option value="newspapers">Newspapers</option>
+            <option value="film-and-videos">Film & Video</option>
+            <option value="notated-music">Notated Music</option>
+            <option value="websites">Websites</option>
+          </select>
         </div>
+        <br>
```
Comment (delizoderek) [inline index.html:46]: IS there a reason you chose to use line breaks instead of setting the margin for the input boxes?

**Review by delizoderek — CHANGES_REQUESTED** [review]
Waiting for question to be answered

File: index.html (hunk 1 above)
Comment (TheHebi) [reply index.html:46]: I started with breaks before trying margins, I can try margins instead if that would be preferred

Comment (delizoderek) [reply index.html:46]: Margins just give more control over the spacing between elements. For now it is fine, mainly wanted to test this feature

**Review by delizoderek — APPROVED** [review]

**Pull request merged**